lnbits/commands.py

```diff
@@ -13,8 +13,40 @@
 from .extension_manager import get_valid_extensions
 
 
-@click.command("migrate")
-def db_migrate():
+@click.group()
+def command_group():
+    """
+    Python CLI for LNbits
+    """
+
+
+@click.command("superuser")
+def superuser():
+    """Prints the superuser"""
+    with open(".super_user", "r") as file:
+        print(f"http://{settings.host}:{settings.port}/wallet?usr={file.readline()}")
+
+
+@click.command("delete-settings")
+def delete_settings():
+    """Deletes the settings"""
+
+    async def wrap():
+        async with core_db.connect() as conn:
+            await conn.execute("DELETE from settings")
+
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(wrap())
+
+
+@click.command("database-migrate")
+def database_migrate():
+    """Migrate databases"""
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(migrate_databases())
```

Comment:
maybe a command group db? so you do `db migrate` or `db versions`. Or just rename to db-migrate and db-verisons to be less verbose